File: src/main/java/no/nav/tjenestepensjon/simulering/nais/NaisLiveness.java
package no.nav.tjenestepensjon.simulering.nais;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class NaisLiveness {

    @GetMapping("/isAlive")
    public HttpStatus isAlive() {
        return HttpStatus.OK;
    }

    @GetMapping("/isReady")
    public HttpStatus isReady() {
        return HttpStatus.OK;
    }
}

File: src/main/kotlin/no/nav/tjenestepensjon/simulering/nais/NaisLiveness.kt
package no.nav.tjenestepensjon.simulering.nais

import org.springframework.http.HttpStatus.OK
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class NaisLiveness {
    @get:GetMapping("/isAlive")
    val isAlive = OK

    @get:GetMapping("/isReady")
    val isReady = OK
}